internal/services/mssql/mssql_virtual_machine_resource_test.go

```diff
@@ -520,8 +512,6 @@ resource "azuread_service_principal" "test" {
 
 resource "azuread_service_principal_password" "test" {
```

Comment:
We should probably try switching this out for an `azuread_application_password` resource since SP passwords are kind of deprecated.

Reply:
Thanks, updated!